CHANGELOG.md

```diff
@@ -1,11 +1,8 @@
 # Changelog
 
-## Deprecated and Scheduled for removal in _Next_ Release
+## Release format
 
-### --privacy-precompiled-address
-Deprecated in 1.5.1
-- CLI option `--privacy-precompiled-address` option removed. This address is now derived, based
-on `--privacy-onchain-groups-enabled`. [\#1222](https://github.com/hyperledger/besu/pull/1222)
+Hyperledger Besu is moving its versioning scheme to [CalVer](https://calver.org/) starting with the 20.10.0 (f.k.a. 1.6.0) release. More information about the specific version of CalVer Besu is using can be found on the [wiki](https://wiki.hyperledger.org/display/BESU/Using+CalVer+for+Besu+Releases). 
```

Comment:
I had to think for a second to figure out f.k.a -> writing in full might be better